docs/src/pages/demos/selects/NativeSelects.js

```diff
@@ -31,10 +30,10 @@ function NativeSelects() {
     name: 'hai',
   });
 
-  const inputLabelRef = React.useRef(null);
```

Comment:
We should perhaps agree a pattern here. I find having ref in the name adds clarity. when viewing the variable in isolation. Admittedly `.current` is a big giveaway, but when it's an actual component ref, I think it has merit. Similarly, having State in the name of state hooks...

Reply:
So far I have seen `*Ref` used for ref objects and for return values from `findDOMNode`. I don't mind adding `*Ref` to return values from `createRef`. But it doesn't make much sense to do something like `ref={ref => (containerRef = React.findDOMNode(ref))}`.

I just think that it doesn't add much value if I append to every variable that gets some `(create|use)Ref` return value the `*Ref` suffix. At that point it only adds noise but no information.

Reply:
I have been suffixing all the ref variables with `Ref`. So no surprises here for me. I would vote for keeping it. No strong opinion here.

Reply:
> I would vote for keeping it.

Keep what? `*Ref` for `(create|use)Ref` or `*Ref` for anything that is a reference to some object?

Reply:
I would vote for the former...

Reply:
> *Ref for (create|use)Ref

@eps1lon Yes. The reason is that it's an objection we are mutating, it's not immutable.

Reply:
> My motivation is that it's an objection we are mutating, it's not immutable. It's an important distinction.

That's not quite correct though. Every write to an instance field is mutating an object (`this`). I guess we can settle on "every var that can be handled by react via `ref` should have the `Ref` suffix"

```js
// valid
<Component ref={ref => (this.someField = ref)} />
<Component ref={this.someRef} />
// invalid
<Component ref{ref => (this.notARef = ref)} />
<Component ref{ref => (this.containerRef = ReactDOM.findDOMNode(ref))} />
```

Reply:
Ok, if you feel it's better without the prefix we can remove it. I was optimizing for class components. In the hook realm, doing:
```jsx
const containerRef = React.useRef();
```
allow us to make a distinction with the over variables.

Reply:
Well in that context it makes sense. I was mostly thinking about usage in `this`. For example
```js
// containerRef is not mutable the same way as `React.createRef` would be mutable.
// The value can be changed but the reference not. Using the `Ref` suffix here is misleading
const { containerRef } = this;

<Component ref={ref => (this.containerRef = ReactDOM.findDOMNode(ref))} />
```